docs/source/_images/Makefile

```diff
@@ -31,14 +31,13 @@ all_tex: $(TEX_FILES:.tex=.pdf) $(TEX_FILES:.tex=.svg)
 
 %.pdf: %.tex
 	cd $(@D) && $(FAKETIME) pdflatex $(<F) --interaction=nonstopmode
+	rm -f $(*).aux $(*).log
```

Comment:
I think the rm was there because of how the docs website was being done and is no longer necessary. It would be better to leave this out entirely and just rely on the gitignore instead of removing the log files.